docs/administrator/security-mode.md

```diff
@@ -0,0 +1,29 @@
+---
+title: security mode
+---
+
+## Security Mode
+
+The Karmada Security Mode document aims to assist users in ensuring the security of their Karmada deployments. This document provides a series of best practices and recommendations to help users protect their Karmada clusters and related resources from potential security risks. It covers various aspects of security, including image signing and secure configurations. By following the recommendations in this guide, users can enhance the security of their Karmada environments and reduce potential security vulnerabilities and attack surfaces. Please note that this document is for reference purposes only, and users should adjust and implement the recommendations according to their specific circumstances and requirements.
+
+### Verify image signature：
+
+Karmada introduced the use of Cosign for image verification starting from version v1.7. For detailed information, please refer to [Verify Artifacts](https://karmada.io/docs/administrator/verify-artifacts).
+
+### Safety Configuration：
+
+#### tls configuration

```

Comment:
Suggestion update: `tls` -> `TLS`

Keep consistent with the content description.